src/Chat/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -805,14 +805,31 @@ Object {
 }
 `;
 
-exports[`Chat should handle messages PRIVMSG HOSTED 1`] = `
+exports[`Chat should handle messages PRIVMSG HOSTED_WITH_VIEWERS 1`] = `
+Object {
+  "_raw": ":jtv!~jtv@jtv.tmi.twitch.tv PRIVMSG #dallas :ronnie is now hosting you for 123 viewers.",
+  "channel": "#dallas",
+  "command": "PRIVMSG",
+  "event": "HOSTED/WITH_VIEWERS",
+  "isSelf": false,
+  "message": "ronnie is now hosting you for 123 viewers.",
+  "numberOfViewers": 123,
```

Comment:
shouldn't `numberOfViewers` be `parameters.numberOfViewers` to be aligned with other events?

Reply:
I thought about this, but because `parameters` is specific to automatically processing the `tags.msgParam*` of `USERNOTICE` events, I decided to make it like [`HOSTTARGET` event](https://github.com/twitch-apis/twitch-js/blob/v2.0.0-beta.12/src/Chat/__tests__/__snapshots__/index.test.js.snap#L199-L212) instead.